src/main/webapp/release_history.html

```diff
@@ -11,57 +11,106 @@
-              <th>Minor</th>
-            </tr>
-            <tr>
-              <th scope="row">19.3.0</th>
-              <th>June 4 2020</th>
-              <th><a href="https://github.com/FirebasePrivate/firebase-android-sdk/tree/M70">M70</th>
-              <th>Major</th>
-            </tr>
-            <tr>
-              <th scope="row">19.2.9</th>
-              <th>June 4 2020</th>
-              <th><a href="https://github.com/FirebasePrivate/firebase-android-sdk/tree/M68">M68</th>
-              <th>Minor</th>
-            </tr>
-            <tr>
-              <th scope="row">19.2.8</th>
-              <th>June 4 2020</th>
-              <th><a href="https://github.com/FirebasePrivate/firebase-android-sdk/tree/M62">M62</th>
-              <th>Minor</th>
-            </tr>
-            <tr>
-              <th scope="row">19.2.7</th>
-              <th>June 4 2020</th>
-              <th><a href="https://github.com/FirebasePrivate/firebase-android-sdk/tree/M58">M58</th>
-              <th>Minor</th>
-            </tr>
-          </tbody>
-        </table>
-      </div>
+    <div class="jumbotron">
+        <h1>Firebase Common</h1>  
+    </div> 
+
+     <div class="row">
+        <div class="col" >
+            <div class="row justify-content-center">
+                <div class="card text-white">
+                    <div class="card">
+                        <div class="card-header">Owner</div>
+                            <div class="card-body">
+                                <p>ashwinraghav@</p>
+                            </div>  
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="col" >
+            <div class="row justify-content-center">
+                <div class="sdk align-items-center d-flex justify-content-center">
+                    <h3>External Name: firebase-commonx</h3>    
+                </div>
+           </div>
+
+            <hr>
+
+           <div class="row justify-content-center">
+                <div class="sdk align-items-center d-flex justify-content-center">
+                    <h3>Latest Version: 19.3.0</h3>    
+                </div>
+           </div>
+        </div>
+
+         <div class="col" >
+            <div class="row justify-content-center">
+                <div class="sdk">
+                    <h4>Library Group: firebase_common</h4>
+                </div>
+           </div>
+
+           <hr>
+
+            <div class="row justify-content-center">
+                <div class="sdk">
+                    <i id="star" class="fa fa-star-o" onclick="favouriteSDK()"></i>
+                </div>
+           </div>
+        </div>
+    </div>
+
+    <hr>
+
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div id="release-history" class="table-responsive">
+                <table class="table table-bordered justify-content-center" style="border: 1px solid #ddd !important;">
```

Comment:
ideally styling should strictly be in css file. refer to https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns